cmd/manager/main.go

```diff
@@ -124,18 +135,18 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Setup all Controllers
-	if err := controller.AddToManager(mgr); err != nil {
-		log.Error(err, "Cannot initialize controller")
-		os.Exit(1)
-	}
-
-	// Setup webhook on master only
-	if _, runWebhookServer := os.LookupEnv("RUN_WEBHOOK_SERVER"); runWebhookServer {
+	// Runs only webhook controllers if it's specified
+	if environment.IsWebhook() {
 		if err := webhook.AddToManager(mgr); err != nil {
```

Comment:
Any special reason not to add controllers in the same approach for all controllers? 
I mean for operator and handlers, we call all controllers, and each controller decides if it should be registered or not, and for webhook case, we only Add the relevant controller here.

Reply:
webhook are not controllers they are different concept at controler-runtime so they have different line.